examples/js/nodes/materials/MeshStandardNodeMaterial.js

```diff
@@ -14,6 +14,10 @@ function MeshStandardNodeMaterial() {
 
 	this.type = "MeshStandardNodeMaterial";
 
+	var options = arguments[ 0 ];
+
+	if ( typeof options === 'object' ) Object.assign( this, options );
```

Comment:
I think we could follow this standard?
https://github.com/donmccurdy/three.js/blob/1ffd8db807e096af16960887c2c086bf3856bc3a/src/materials/MeshStandardMaterial.js#L56